gensim/test/test_data/mihalcea_tarau.txt

```diff
@@ -1,11 +1,11 @@
-AP880911-0016
-AP-NR-09-11-88 0423EDT r i
-BC-HurricaneGilbert 09-11 0339
-BC-Hurricane Gilbert,0348
-Hurricane Gilbert Heads Toward Dominican Coast
-By RUDDY GONZALEZ
-Associated Press Writer
-SANTO DOMINGO, Dominican Republic (AP)
+AP880911-0016.
+AP-NR-09-11-88 0423EDT r i.
```

Comment:
Without dots, this doesn't work? You think that this is a good solution?

Reply:
Not really, but this was reported in #1575 and I created this PR as an improvement in performance over #1577.

I do believe that we should at least write about this behavior in the docstring.